pallets/crowdloan-claim/src/lib.rs

```diff
@@ -417,120 +357,68 @@ pub mod pallet {
     // Additional argument to specify the metadata to use for given type
     #[pallet::metadata(T::AccountId = "AccountId")]
     pub enum Event<T: Config> {
-
-        /// A new contributor was registered, being elligible for future reward payout claims
-        ///
-        /// A registered contributor proves her/his identity to the parachain,
-        /// given a relay chain account id, a parachain account id and a
-        /// signature, in order to prove her/his native identity.
-        /// \[relaychain_account_id, parachain_account_id\]
-        ContributorRegistered(T::RelayChainAccountId, ParachainAccountIdOf<T>),
-
         /// Event triggered when a reward has already been processed.
         /// \[who, amount\]
         ClaimAlreadyProcessed(T::RelayChainAccountId, ContributionAmountOf<T>),
```

Comment:
Do we need this event? Can we error out when the contributor already claimd the reward ?

Reply:
Yes, this is better, will change that.